_drafts/wework-com-is-going-static.markdown

```diff
@@ -0,0 +1,111 @@
+---
+layout:       post
+title:        Why WeWork.com uses a static generator and why you should too
+author:       ramin_bozorgzadeh
+summary:      Back when the web first started, things were a lot simpler. Most websites were made up of static html pages and there weren't a lot of moving parts. This is the story of how wework.com went from a monolithic "web app" to a statically generated site, and why ...
+image:        //res.cloudinary.com/wework/image/upload/c_fill,f_auto,g_faces,h_800,w_1000/v1448734984/engineering/wework-com-is-going-static.jpg
+categories:   engineering
+---
+
+### What is this "static" business all about?
+
+Static site generators have become a hot topic in the past few years. There are [whole sites](https://www.staticgen.com/) dedicated to tracking and rating them. Don't take my word
```

Comment:
A little awkward breaking of sentences. Maybe...

Over the past few years, static site generators have become a hot topic, so much so that [entire sites](https://www.staticgen.com/) have emerged dedicated to tracking and rating them.